### DISCUSSION TITLE
Make Flutter/GUI as optional build feature

### DISCUSSION THREAD

**Discussion by T-256** [rustdesk/rustdesk#9287] [General]
Currently Rustdesk compiles flutter deps into binary to provide GUI. Since we could do most things via CLI, is it possible publish CLI-only releases? (it will run by `rustdesk --server`).

FWIW, I think the final binary size will drop dramatically! (I didn't look at codebase, so I don't know even it's possible or not)

**Comment by basilgello** [comment]
It should be possible to use headless mode with Pro subscription and custom client.

**Reply by T-256** [reply]
> It should be possible to use headless mode with Pro subscription and custom client.

I don't mean headless mode which is currently available for hosts without screen.
I mean user interface, which is currently has both gui and cli, since we don't need gui, is it possible build rustdesk without flutter? then when running rustdesk it won't load graphical window to interact with. so, then it'd enough to run `rustdesk --server` and connect to it via output of `rustdesk --get-id`.

**Reply by basilgello** [reply]
With current source code base - no. Either sciter or flutter. But I can do that customization for you on contract basis. You can reach me on email used for commit sign-offs.
